Socket.IO-Client-Swift.xcodeproj/project.pbxproj

```diff
@@ -1316,7 +1315,6 @@
 				PRODUCT_NAME = SocketIO;
 				SDKROOT = appletvos;
 				SKIP_INSTALL = YES;
-				TARGETED_DEVICE_FAMILY = "1,2";
```

Comment:
This needs to be set to `3` (see https://github.com/jspahrsummers/xcconfigs/blob/master/tvOS/tvOS-Base.xcconfig#L15)